extensions/timer/runtime/src/main/resources/META-INF/quarkus-extension.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Camel Quarkus Timer",
+  "labels": [
+    "integration",
+    "camel",
+    "timer"
```

Comment:
We are using scheduling as labels for these kind of timer components
https://github.com/apache/camel/blob/master/components/camel-timer/src/main/java/org/apache/camel/component/timer/TimerEndpoint.java#L42